include/ratelimit_manager.h

```diff
@@ -148,14 +155,36 @@ class RateLimitManager
         // Clear all rules
         void clear_all();
 
+        // Set store
+        Option<bool> init(Store* store);
+
     private:    
 
         RateLimitManager() {
             rate_limit_request_counts.capacity(10000);
         }
 
+        // Store for rate limit rules
+        Store *store;
+
+        // Using a $ prefix so that these meta keys stay above record entries in a lexicographically ordered KV store
+
+        // Prefix for rate limit rules
+        static constexpr const char* RULES_NEXT_ID = "$KRLN";
```

Comment:
The `K` is superfluous. Can just begin everything with `$RL...` instead (RL for RateLimit component). So:

- $RLN
- $RLRP
- $RLBN
- $RLBP